src/test/ui/suggestions/issue-51244.nll.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0594]: cannot assign to data in a `&` reference
+  --> $DIR/issue-51244.rs:13:5
+   |
+LL |     let ref my_ref @ _ = 0;
+   |         -------------- help: consider changing this to be a mutable reference: `&mut ef my_ref @ _`
```

Comment:
This suggestion is wrong

Reply:
Blocked by https://github.com/rust-lang/rust/pull/51612

Reply:
@pietroalbini block occurs here.